lib/file_database.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'json'
+
+class FileDatabase
+  def initialize(file_dir)
+    @file = File.open(file_dir, 'r')
+  end
+
+  def all
+    file.rewind
+    convert_json(file.read)
+  end
+
+  def database_empty?
+    all.any?
```

Comment:
this would be `all.empty?` no?

otherwise you are saying the database is empty if there are any contacts, 
and is not empty when there are no contacts

Reply:
Doh!! how silly of me!